web/app.php

```diff
@@ -2,14 +2,16 @@
 
 use Symfony\Component\HttpFoundation\Request;
 
-$loader = require_once __DIR__.'/../app/bootstrap.php.cache';
+$loader = require __DIR__.'/../app/autoload.php';
```

Comment:
Can we do this already (< php 7) ?

Reply:
Yes I think we can

See https://symfony.com/blog/new-in-symfony-3-3-deprecated-the-classloader-component